gateway/src/lib.rs

```diff
@@ -571,13 +571,15 @@ pub mod tests {
                     cch_container_limit: 1,
                     soft_container_limit: 2,
                     hard_container_limit: 3,
+
+                    // Allow access to the auth on the host
+                    extra_hosts: vec!["host.docker.internal:host-gateway".to_string()],
```

Comment:
I am just seeing this, why do we need it?

Reply:
Don't know, I just moved it

Reply:
Fyi, this is for tests. Since the gateway is not started in a container, auth and the other service fail to reach it in the tests. So this makes docker resolve the `host-gateway` lookups to the host machine correctly.